src/build.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const core = require('@actions/core')
+const { exec } = require('@actions/exec')
+const parseReleaseMetadata = require('./utils/parseReleaseMetadata')
+const { logError } = require('./log')
+
+module.exports = async function ({ context, inputs }) {
+  let releaseMetadata
+
+  try {
+    releaseMetadata = parseReleaseMetadata(context.payload.pull_request)
```

Comment:
just a question. how do we know that context.payload.pull_request is populated?

Reply:
I think it will always be there since this step will only run for pull requests as defined herr but I'm not entirely sure.
https://github.com/nearform/optic-release-automation-action/blob/e60e3dd3b4f71f1d7ee1746600e634c6de58f49d/action.yml#L102